.github/workflows/docs.yaml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches: [main, docs-deployment]
```

Comment:
Please remove `docs-deployment` (I'm guessing this was for testing?).

Reply:
Yep I can't run Github actions properly locally so I have to add my dev branch to test my changes, I'll make sure to remove it once I can confirm it's working well.

Reply:
It's removed now